Floofbot/Modules/Utilities.cs

```diff
@@ -49,16 +49,20 @@ public async Task UserInfo(IGuildUser usr = null)
 
             string avatar = "https://cdn.discordapp.com/attachments/440635657925165060/442039889475665930/Turqouise.jpg";
 
+            // Get user's Discord joining date and time, in UTC
+            string discordJoin = user.CreatedAt.ToUniversalTime().ToString("dd\\/MMM\\/yyyy \\a\\t H:MM \\U\\T\\C");
+            // Get user's Guild joining date and time, in UTC
+            string guildJoin = user.JoinedAt?.ToUniversalTime().ToString("dd\\/MMM\\/yyyy \\a\\t H:MM \\U\\T\\C");
+
             if (user.AvatarId != null)
                 avatar = user.GetAvatarUrl(ImageFormat.Auto, 512);
 
-            string infostring = $"👥 **User info for:** {user.Mention}\n";
```

Comment:
I think I personally prefer it to say the old version than the new one here

Reply:
Yes, it'll be more clear that way